Games/types/Mafia/roles/cards/GuardianAngel.js

```diff
@@ -0,0 +1,36 @@
+const Card = require("../../Card");
+const { PRIORITY_EFFECT_GIVER_DEFAULT } = require("../../const/Priority");
+
+module.exports = class GuardianAngel extends Card {
+  constructor(role) {
+    super(role);
+
+    this.meetings = {
+      Guardian: {
+        states: ["Night"],
+        flags: ["voting"],
+        shouldMeet: function () {
+          return !this.data.angelTarget;
+        },
+        action: {
+          labels: ["guardian", "hidden", "absolute"],
+          priority: PRIORITY_EFFECT_GIVER_DEFAULT,
+          run: function () {
+            this.actor.role.data.angelTarget = this.target;
```

Comment:
@Golbolco this code lets angel choose their target.... feels like angel can delay choosing target until late game and then win...

to prevent angel target n1 dying we can let the angel get a target on daystart.....